pyproject.toml

```diff
@@ -16,7 +16,7 @@ flake8 = "*"
 pytest-cov = "*"
 isort = "^5.10.1"
 mypy = ">=1"
-snowflake-connector-python = "^2.7.11"
+snowflake-connector-python = "^3.0.1"
```

Comment:
was not sure about upgrading the connector, had some local issues with the old.
what do you think?

Reply:
change it to `>=2.7.11`, to not break anything